saas_demo/models/saas_operator.py

```diff
@@ -25,6 +25,10 @@ class SAASOperator(models.Model):
     def is_local(self):
         return any((r.type == 'local' for r in self))
 
+    @api.model
+    def test_module_installed(self):
+        return self.env['ir.module.module'].search([('name', '=', 'saas_demo_test')])
```

Comment:
The method name should be is_test -- we may change the way of checking test in future.

Reply:
Also, the method should not be attached to saas.operator. I'd prefer isolated python file (like xmlrpc)